packages/codelabs/src/basics/web-components.md

```diff
@@ -222,102 +222,45 @@ If we run this code in the browser, the element should turn blue when clicked on
+</aside>
 
-LitElement is written and distributed as ES Modules, this means we can import it using the browser's native module loader. Let's create a module script, and import `LitElement` from a CDN:
+Let's create a `<template>` inside the body of our previous excercise.
 
 ```html
-<!DOCTYPE html>
-
-<html>
-  <body>
-    <script type="module">
-      import { LitElement } from 'https://unpkg.com/lit-element?module';
-    </script>
-  </body>
-</html>
+<template></template>
 ```
 
-Make sure you add `type="module"` to the script tag!
-
-Next, we need to define our class. Instead of extending `HTMLElement`, we are now extending `LitElement`. `LitElement` extends `HTMLElement` already, so we're still creating an actual custom element:
-
-```js
-class WebComponentApis extends LitElement {
-  connectedCallback() {
-    super.connectedCallback();
-    console.log('lit element connected');
-  }
-}
-
-customElements.define('web-component-apis', WebComponentApis);
-```
+Next, we'll move the inner HTML of `<cool-heading>` to the template:
 
 ```html
-<web-component-apis></web-component-apis>
-```
-
-If you run this in the browser you should see `lit element connected` logged to the terminal.
-
-Now that we have our element based on LitElement, we can start adding our template. lit-html works by writing HTML inside of template literals. These are a type of strings which can span multiple lines, ideal for writing HTML:
-
-```js
-const template = `
- <h1>Hello world</h1>
-`;
-```
-
-In order to create an actual lit-html template, we need to prefix the template literal with a special HTML tag:
-
-```js
-import { html } from 'https://unpkg.com/lit-element?module';
-
-const template = html`
-  <h1>Hello world</h1>
-`;
+<template>
+  <h1>Hello world!</h1>
+</template>
 ```
 
-This is a native browser feature called [tagged template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates). The `html` tag is just a function that gets called with information about template literal it is attached to. We won't go into the details of how it works exactly, but by using this syntax `lit-html` we can very efficiently create templates and update only the parts that change when re-rendering.
-
-Most popular IDEs support syntax highlighting of HTML inside template literals, but for some you might need to install a plugin. [See our IDE section](https://open-wc.org/developing/ide.html#visual-studio-code) to learn more about that.
+Notice that "Hello World" is not rendered anymore. As the HTML is now a child of a template, the browser interprets it as a HTML Fragment, which is not part of the page's DOM.
```

Comment:
```suggestion
Notice that "Hello World" is no longer rendered on the page. This is because the content of a `<template>` elements is inert. Nothing inside it is displayed, images don't get downloaded and scripts don't run.
```

Reply:
should we mention/explain the term DocumentFragment? You can see it in the browser tools while inspecting.
And it is inert because it’s not directly in the DOM, right? :-)

Reply:
should we mention/explain the term DocumentFragment? You can see it in the browser tools while inspecting.
And it is inert because it’s not directly in the DOM, right? :-)

Reply:
should we mention/explain the term DocumentFragment? You can see it in the browser tools while inspecting.
And it is inert because it’s not directly in the DOM, right? :-)

Reply:
I'd like to not introduce terminology people don't need to know, I think it's clearer like this but open for suggestions on my suggestion :P